extensions/default/src/DicomWebDataSource/index.js

```diff
@@ -15,6 +15,7 @@ import {
   retrieveStudyMetadata,
   deleteStudyMetadataPromise,
 } from './retrieveStudyMetadata.js';
+import StaticWadoClient from './utils/StaticWadoClient.js';
```

Comment:
That static wado stuff includes a bunch of search functionality to allow sub-selecting the returned studies results.  It may well be something we want to have supported when queries are done against servers which are less than robust in terms of search functionality.  Alternatively, it could simply be left out for now.